sheet.go

```diff
@@ -869,6 +869,10 @@ func (f *File) searchSheet(name, value string, regSearch bool) ([]string, error)
 //
 // - No footer on the first page
 //
+
```

Comment:
Remove blank lines, please read the [CONTRIBUTING](https://github.com/360EntSecGroup-Skylar/excelize/blob/master/CONTRIBUTING.md).